stan/math/prim/mat/prob/ordered_logistic_lpmf.hpp

```diff
@@ -80,7 +75,7 @@ typename return_type<T_loc, T_cut>::type ordered_logistic_lpmf(
   } else if (y == K) {
     logp_n -= log1p_exp(c[K - 2] - lambda);
   } else {
-    logp_n += log_inv_logit_diff(c[y - 2] - lambda, c[y - 1] - lambda);
+    logp_n += log_inv_logit_diff(lambda - c[y - 2], lambda - c[y - 1]);
```

Comment:
Why are these changing orders?  It changes the sign of the arguments.  Dos it not matter for this function or is it a different function?

Reply:
Right, I should have made a note here. The previous (inline) implementation of the function evaluated to:
```
log_inv_logit_diff(x,y) = log(inv_logit(y) - inv_logit(x))
```
Which isn't consistent with other two-argument functions (like ```log_diff_exp```). The new (standalone) function evaluates to:
```
log_inv_logit_diff(x,y) = log(inv_logit(x) - inv_logit(y))
```
So I had to flip the sign of the arguments to keep the result the same.